docs/pages/enroll-resources/kubernetes-access/faq.mdx

```diff
@@ -33,3 +33,8 @@ Check out the [Kubernetes Service Discovery
 Guide](../auto-discovery/kubernetes/kubernetes.mdx) for more
 documentation and examples.
 
+## Does Teleport work with Kubernetes desktop UI applications?
+
+Yes, Teleport generates a kubeconfig file (default `~/.kube/config`) when a user
+logins to a Kubernetes cluster. GUI tools such as Lens and other applications can 
```

Comment:
```suggestion
logs in to a Kubernetes cluster. GUI tools such as Lens can 
```
- Grammar fix ("logins to" is incorrect)
- Removing redundant wording ("such as ... and other applications")